EIPS/eip-3549.md

```diff
@@ -0,0 +1,196 @@
+---
+eip: 3549
+title: ERC-721 Metadata Signature Standard
+author: Clay Graham <claytantor@gmail.com>
```

Comment:
```suggestion
author: Clay Graham (@claytantor)
```
We require at least one author list a github handle so they can be tagged for notifications.